### ISSUE TITLE
DateTimePicker - Unable to set tab index for button

### ISSUE THREAD

**Issue by sebastiancaso** [jquense/react-widgets#1049]
Button on DateTimePicker has tab index hard coded to -1.  As a result keyboard users cannot access it.  Are there any plans or thoughts against making it configurable?
